recipes/statista/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "statista" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MAfarrag/statista/archive/{{ version }}.tar.gz
+  sha256: 6a4b4578e54ad4b4fdff803ff9041a536f744871f7fa926883649319b6f1e52b
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python  >=3.7.1,<3.10
+    - pip >=21.3.1
+  run:
+    - python >=3.7.1,<3.10
+    - pip >=21.3.1
+    - matplotlib-base >=3.4.2,<3.6.0
+    - pandas >=1.3.2,<1.4.3
+    - scipy >=1.8.1
+    - scikit-learn >=0.24.2,<1.1.2
+    - loguru >=0.5.3
+    - pytest >=6.2.5
+    - pytest-cov >=2.12.1
```

Comment:
depend on what you want, but it is very likely that you don't need `pytest`  to run your package. It seems to be a testing dependency. You might as well remove it from here.